frontend/src/lib/components/CommandBar/SearchTabs.tsx

```diff
@@ -0,0 +1,23 @@
+import { useValues } from 'kea'
+
+import { searchBarLogic } from './searchBarLogic'
+import SearchBarTab from './SearchBarTab'
+import { ResultType } from './types'
+
+const SearchTabs = (): JSX.Element | null => {
+    const { searchResponse, activeTab } = useValues(searchBarLogic)
+
+    if (!searchResponse) {
+        return null
+    }
+
+    return (
+        <div className="flex items-center border-t">
```

Comment:
I think we could get rid of the `isFirst` prop below if we use child spacing:

```suggestion
        <div className="flex items-center border-t space-x-3 px-2">
```